### PR TITLE
Bedroom Crew~

### PR DESCRIPTION
**Pull request by Funce**
## About The Pull Request
Don't mind the title, its just a meme.
Renames trusted crew to VETERAN crew in the code because, yeah a name change!

## Why It's Good For The Game
Gandalf told me to

### CODE REVIEW

**Comment by hugmeorelse** [comment]
I hate this

**Comment by Cheshify** [comment]
Bad PR I am going to fucking scream 👍

File: modular_skyrat/modules/customization/modules/client/preferences.dm [hunk 1]
```diff
@@ -1383,8 +1383,8 @@ GLOBAL_LIST_INIT(food, list(
 			if(is_banned_from(user.ckey, rank))
 				HTML += "<font color=red>[rank]</font></td><td><a href='?_src_=prefs;bancheck=[rank]'> BANNED</a></td></tr>"
 				continue
-			if(job.trusted_only && !is_trusted_player(user.client))
-				HTML += "<font color=red>[rank]</font></td><td><font color=red> \[NOT TRUSTED\]</font></td></tr>"
+			if(job.veteran_only && !is_veteran_player(user.client))
+				HTML += "<font color=red>[rank]</font></td><td><font color=red> \[NOT VETERAN\]</font></td></tr>"
```
Comment (FlamingLily) [inline modular_skyrat/modules/customization/modules/client/preferences.dm:1387]:
```suggestion
				HTML += "<font color=red>[rank]</font></td><td><font color=red> \[NOT BEDROOM CREW\]</font></td></tr>"
```
Think you made a typo

**Comment by Funce** [comment]
I'm waiting to merge this, because I need to change the config when I do.

**Pull request merged**